src/assets/drizzle/styles/components/_icon-link.scss

```diff
@@ -0,0 +1,12 @@
+.#{$app-namespace}-c-icon-link {
+  fill: $white;
+  display: inline-block;
+}
+
+.#{$app-namespace}-c-icon-link:hover {
+  fill: $red-light;
```

Comment:
We don't want all icons that are links to have a hover fill of $red-light. You only want these styles for the footer links.

Reply:
Done!